lib/grape/validations/types.rb

```diff
@@ -22,56 +13,37 @@ module Validations
     # and {Grape::Dsl::Parameters#optional}. The main
     # entry point for this process is {Types.build_coercer}.
     module Types
-      # Types representing a single value, which are coerced.
-      PRIMITIVES = [
```

Comment:
While this is for "internal use" it was available as a public constant, so it might be a breaking change for someone.